### ISSUE TITLE
Overflow breaks currency, warn system broke.

### ISSUE THREAD

**Issue by witt2witty** [DanielOhanessian/Onessia-Bot#2]

**Comment by AdamS1816** [comment]
I agree with first statement: but we do use the Dyno warn system: it’s simpler and restricted to certain people.

**Comment by XyplhTheGlyph** [comment]
> Super easy to write a decent warn system
>Prefers to use shitty discord bot
lmao

**Comment by AdamS1816** [comment]
Please elaborate- how could he improve on the current warn system? What could he do to make it better? Dyno is mostly likely the best one out there.

**Comment by XyplhTheGlyph** [comment]
![screen shot 2018-01-31 at 7 15 44 pm](https://user-images.githubusercontent.com/32025056/35654454-2af36184-06bb-11e8-8eef-61ebe99b15dc.png)

**Comment by XyplhTheGlyph** [comment]
Points, time that goes away, and not a shitty bot

**Comment by AdamS1816** [comment]
It doesn’t look like that bot makes point go away over time. And plus it automatically bans you if you go over a certain amount. We would like to review said persons warning b4 banning or other form of punishment. I don’t think Daniel is capable of making that complex of a warning system.

**Comment by XyplhTheGlyph** [comment]
You can enable or disable that function lol. Cuthy is an example of a worthwhile bot.

**Comment by AdamS1816** [comment]
I will look into that bot. Maybe Daniel can make a warning system that complex. We appreciate your input.

**Comment by XyplhTheGlyph** [comment]
lmao do you even know the link?

**Comment by AdamS1816** [comment]
No. We appreciate you providing us with the link. Thank you for your time

**Comment by AdamS1816** [comment]
After further review, we have found that Cuthbert is not an acceptable bot for The Kingdom. Thank you for your suggestions.
